pyaedt/edb_core/EDB_Data.py

```diff
@@ -254,6 +254,28 @@ def init_vals(self):
 
     @aedt_exception_handler
     def update_layer_vals(self, layerName, newLayer, etchMap, materialMap, fillMaterialMap, thicknessMap, layerTypeMap):
+        """Update layer properties.
+        
+        Parameters
+        ----------
+        layerName :
+        
+        newLayer :
+        
+        materialMap :
+        
+        fillMaterialMap :
+        
+        thicknessMap :
+        
+        layerTypeMap :
+        
+        Returns
+        -------
+        type
+            Layer object.
```

Comment:
@maxcapodi78 I am not very familiar with EDB, so do you know what the types of these variables are?